.github/workflows/tag-validation.yml

```
@@ -0,0 +1,176 @@
+            echo "number=${{ github.event.pull_request.number }}" >> $GITHUB_OUTPUT
+          else
+            echo "number=0" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Run TAG validation (info mode)
+        if: github.event_name == 'push' || github.event.pull_request.draft == false
+        id: validate_info
+        continue-on-error: true
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2028:info:9:8: echo may not expand escape sequences. Use printf

(shellcheck)